pkg/controllers/dynakube/extension/secret.go

```diff
@@ -0,0 +1,83 @@
+package extension
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/dttoken"
+	k8ssecret "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/secret"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	eecTokenKey         = "eec-token"
```

Comment:
in my PR i've created consts.go similar to other reconcilers, please add also this file and move it there.

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/3487/files#diff-ffc8830719c7c7221619364b5ad1ee791546f72852e019535f4ff8b59c44f4a0

Reply:
[Fixed here](https://github.com/Dynatrace/dynatrace-operator/pull/3485/commits/ba5a535b2e26209d7c415d79cdd93824001c3f68)